doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -134,7 +134,7 @@ PyArray_Type
     does not own its own memory, then base points to the Python object
     that owns it (perhaps another array object), 2) If this array has
     the (deprecated) :c:data:`NPY_ARRAY_UPDATEIFCOPY` or
```

Comment:
Might be clearer if the `1)` and `2)` are made into a formatted list (no numbers required, the order of the points is not important)

Reply:
Blank needed before the first item in the list and before the following sentence "When ..."